.github/workflows/ci.yml

```diff
@@ -124,6 +124,13 @@ jobs:
         env:
           POETRY_LIBPARSEC_BUILD_STRATEGY: no_build
 
+      - name: Additional Check
+        run: |
+          set -ex
+          rustup target add wasm32-unknown-unknown
+          cargo check --no-default-features --features use-sodiumoxide
+          cargo check --target wasm32-unknown-unknown
```

Comment:
cargo check could use the build artifact to be faster.
So move it to `test-rust-matrix` like for the task `pre-commit clippy`

Reply:
Is it better ?

Reply:
Yes because we could use the cache

Reply:
But the important part is to measure

Reply:
It takes 3mins to check but we win 2mins in rust test (so 1min remains)
And it was approximately ~ 2mins